frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -78,13 +79,21 @@ class ListDropdown_ extends React.Component {
           unsortedList,
         );
       });
-
       const sortedList = {};
       _.keys(unsortedList)
         .sort()
         .forEach((key) => {
           sortedList[key] = unsortedList[key];
         });
+
+      if ((selectedKey === undefined || !_.isEmpty(selectedKey)) && selectFirst) {
+        const keys = _.keys(sortedList);
+        if (keys.length > 0) {
```

Comment:
```suggestion
        if (keys.length) {
```

Reply:
I prefer to keep it as keys.length > 0 for a bit for readability as a numerical value instead of just checking truthy and falsey.  Let me know if you prefer it this way.